server/models/db/workerTimeslot.js

```diff
@@ -1,8 +1,9 @@
 const _ = require('lodash');
 
 const { db } = require('../../lib/connection');
+
 const Tables = require('../../constants/tables.json');
-const TimeslotStatus  = require('../../constants/timeslot-status.json');
+const TimeslotStatus  = require('../../constants/timeslotStatus.json');
```

Comment:
Please note that the timeslot status constants file was renamed to lowerCamelCase to be consistent with the naming of the other constant files.

Reply:
Good idea to keep naming consistent!